Demos/libraries/mymath.h

```diff
@@ -1 +1,3 @@
-double sinc(double);
+#include <Python.h>
+
+Py_EXPORTED_SYMBOL double sinc(double);
```

Comment:
I still think that a C library example should not be using `Python.h` and `Py_EXPORTED_SYMBOL`. It's a C library, not a Python extension.

Reply:
So, you mean to say you have a very strong emphasis on how to craft external C libraries, without Python provisions?
I understand, it just looked to me that a gentle suggestion to study what is available in Python ( and seeing how it supports portable code ) would help.
And it does look aesthetically pleasing to an eye a bit better than lines of #define(s) I had earlier.

Reply:
> you have a very strong emphasis on how to craft external C libraries

No, I just think we shouldn't give wrong impressions in our documentation.